CHANGELOG.md

```diff
@@ -0,0 +1,115 @@
+# react-uikit change log
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [0.6.2] - 2016-08-24
```

Comment:
We should add a Unreleased section while add it, mentioning the addition of a changelog as well as the Notifications component 😄